2021-12-13 19:49:35,066 INFO    MainThread:14388 [wandb_setup.py:_flush():71] setting env: {}
2021-12-13 19:49:35,067 INFO    MainThread:14388 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-13 19:49:35,067 INFO    MainThread:14388 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_194935-1vp75v90\logs\debug.log
2021-12-13 19:49:35,067 INFO    MainThread:14388 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_194935-1vp75v90\logs\debug-internal.log
2021-12-13 19:49:35,067 INFO    MainThread:14388 [wandb_init.py:init():404] calling init triggers
2021-12-13 19:49:35,067 INFO    MainThread:14388 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-13 19:49:35,069 INFO    MainThread:14388 [wandb_init.py:init():449] starting backend
2021-12-13 19:49:35,069 INFO    MainThread:14388 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-12-13 19:49:35,073 INFO    MainThread:14388 [backend.py:ensure_launched():199] starting backend process...
2021-12-13 19:49:35,158 INFO    MainThread:14388 [backend.py:ensure_launched():204] started backend process with pid: 1912
2021-12-13 19:49:35,158 INFO    MainThread:14388 [wandb_init.py:init():458] backend started and connected
2021-12-13 19:49:35,160 INFO    MainThread:14388 [wandb_init.py:init():519] updated telemetry
2021-12-13 19:49:35,180 INFO    MainThread:14388 [wandb_init.py:init():549] communicating current version
2021-12-13 19:49:35,693 INFO    MainThread:14388 [wandb_init.py:init():554] got version response 
2021-12-13 19:49:35,693 INFO    MainThread:14388 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-13 19:49:35,916 INFO    MainThread:14388 [wandb_init.py:init():592] starting run threads in backend
2021-12-13 19:49:36,015 INFO    MainThread:14388 [wandb_run.py:_console_start():1816] atexit reg
2021-12-13 19:49:36,017 INFO    MainThread:14388 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-12-13 19:49:36,017 INFO    MainThread:14388 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-12-13 19:49:36,020 INFO    MainThread:14388 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-13 19:49:36,020 INFO    MainThread:14388 [wandb_init.py:init():619] run started, returning control to user process
2021-12-13 19:49:36,020 INFO    MainThread:14388 [wandb_config.py:__setitem__():144] config set learning_rate = 0.001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x00000202035A3B20>>
2021-12-13 19:49:36,021 INFO    MainThread:14388 [wandb_run.py:_config_callback():962] config_cb learning_rate 0.001 None
2021-12-13 19:49:36,021 INFO    MainThread:14388 [wandb_config.py:__setitem__():144] config set batch_size = 32 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x00000202035A3B20>>
2021-12-13 19:49:36,021 INFO    MainThread:14388 [wandb_run.py:_config_callback():962] config_cb batch_size 32 None
2021-12-13 19:49:36,022 INFO    MainThread:14388 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-13 19:49:36,023 INFO    MainThread:14388 [wandb_run.py:_restore():1758] restore
2021-12-13 19:49:37,029 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 19:49:37,295 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 19:49:37,953 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 2755
}

2021-12-13 19:49:38,070 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 2781
}

2021-12-13 19:49:38,178 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 2781
}

2021-12-13 19:49:38,287 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2781
  total_bytes: 2781
}

2021-12-13 19:49:38,394 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2781
  total_bytes: 2781
}

2021-12-13 19:49:38,502 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2781
  total_bytes: 2781
}

2021-12-13 19:49:38,611 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2781
  total_bytes: 2781
}

2021-12-13 19:49:38,721 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2781
  total_bytes: 2781
}

2021-12-13 19:49:38,830 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2781
  total_bytes: 2781
}

2021-12-13 19:49:38,989 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2781
  total_bytes: 2781
}

2021-12-13 19:49:39,546 INFO    MainThread:14388 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2781
  total_bytes: 2781
}
local_info {
}

2021-12-13 19:49:40,614 INFO    MainThread:14388 [wandb_run.py:_append_files():2184] logging synced files
